src/pages/bulkImport/BulkReportForm.jsx

```diff
@@ -25,10 +26,44 @@ import useBulkImportFields from './utils/useBulkImportFields';
 import {
   validateMinMax,
   validateIndividualNames,
-  validateAssetStrings,
+  validateStrings,
 } from './utils/flatfileValidators';
 
-async function onRecordChange(record, recordIndex, filenames) {
+function vectorizeCustomMultiselect(
+  sightingData,
+  customMultiselects,
+) {
+  const modifiedSightingData = sightingData; // it looks like the original sightingData is also getting mutated by the below. Advice on how to avoid that?
+
+  modifiedSightingData.forEach((sighting, sightingIdx) => {
+    customMultiselects.forEach(element => {
```

Comment:
These will be small, so I figured that it was ok to not find a more efficient way (assuming that's possible).